packages/mdc-select/mdc-select.scss

```diff
@@ -177,3 +177,66 @@
     }
   }
 }
+
+/* postcss-bem-linter: define multi-select */
+
+/* stylelint-disable plugin/selector-bem-pattern */
```

Comment:
I'd rather this be disable in a more local place, rather than at the top of the entire component selector, even if that means using it in multiple places